CMakeLists.txt

```diff
@@ -431,52 +456,46 @@ set(LIB_LIST)
 #
 # ZLIB (Only needed for deflate extensions).
 #
-if (NOT LWS_WITHOUT_EXTENSIONS)
-	if (WIN32 AND NOT LWS_USE_EXTERNAL_ZLIB)
-		message("Using included Zlib version")
-
-		# Compile ZLib if needed.
-		set(WIN32_ZLIB_PATH "win32port/zlib")
-		set(ZLIB_SRCS
-			${WIN32_ZLIB_PATH}/adler32.c
-			${WIN32_ZLIB_PATH}/compress.c
-			${WIN32_ZLIB_PATH}/crc32.c
-			${WIN32_ZLIB_PATH}/deflate.c
-			${WIN32_ZLIB_PATH}/gzclose.c
-			${WIN32_ZLIB_PATH}/gzio.c
-			${WIN32_ZLIB_PATH}/gzlib.c
-			${WIN32_ZLIB_PATH}/gzread.c
-			${WIN32_ZLIB_PATH}/gzwrite.c
-			${WIN32_ZLIB_PATH}/infback.c
-			${WIN32_ZLIB_PATH}/inffast.c
-			${WIN32_ZLIB_PATH}/inflate.c
-			${WIN32_ZLIB_PATH}/inftrees.c
-			${WIN32_ZLIB_PATH}/trees.c
-			${WIN32_ZLIB_PATH}/uncompr.c
-			${WIN32_ZLIB_PATH}/zutil.c
-		)
-
-		# Create the library.
-		add_library(ZLIB STATIC ${ZLIB_SRCS})
-
-		# Set the same variables as find_package would.
-		set(ZLIB_INCLUDE_DIRS ${WIN32_ZLIB_PATH})
-		get_property(ZLIB_LIBRARIES TARGET ZLIB PROPERTY LOCATION)
-		set(ZLIB_FOUND 1)
-	else()
+if (LWS_WITH_ZLIB)
+	if (LWS_USE_BUNDLED_ZLIB)
```

Comment:
You changed the logic here, so the default Windows build breaks.

On windows the default should be to use the bundled zlib